modules/zero-trust-manager-spire-server-config.adoc

```diff
@@ -24,41 +24,93 @@ You can configure the `SpireServer` custom resource (CR) to deploy and configure
 +
 [source,yaml]
 ----
-apiVersion: operator.openshift.io/v1alpha1
+aapiVersion: operator.openshift.io/stable-v1
 kind: SpireServer
 metadata:
-  name: cluster
+ name: cluster
```

Comment:
```suggestion
  name: cluster
```

Reply:
Could you please fix the YAML indentations as per example provided by @anirudhAgniRedhat: https://github.com/openshift/openshift-docs/pull/103762#discussion_r2623674861